### PR TITLE
WIP:no CI for irrevelant files

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

/hold

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -2,7 +2,14 @@
     name: kubernetes-sigs/cluster-api-provider-openstack
     check:
       jobs:
-        - cluster-api-provider-openstack-current-acceptance-test-v1.18
+        - cluster-api-provider-openstack-current-acceptance-test-v1.18:
```
Comment (sbueringer) [inline .zuul.yaml:5]: I would really like to replace OpenLab with Prow. 
As soon as I have [this PR](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/759) green and we have the v1alpha4 and this PR merged there is no need to keep OpenLab:

Comment (jichenjc) [reply .zuul.yaml:5]: it's different purpose, I want to avoid run CI for non-related test cases..

Comment (sbueringer) [reply .zuul.yaml:5]: I know, but I will delete this file in my PR because this file is for OpenLab, it doesn't affect our Prow builds.

Comment (jichenjc) [reply .zuul.yaml:5]: ok, I got it , so I can wait for your PR ready then work on that part

Comment (sbueringer) [reply .zuul.yaml:5]: Yeah you basically have to do that change in the Prow test-infra repo (where I added the new jobs).
Problem is that Prow does not allow something like this as far as I know. You can say "only run this job when those files change" but you cannot say "don't run this job when only those files change).

**Comment by sbueringer** [comment]
As the .zuul.yaml has been deleted with OpenLab I'll close this PR
/close

**Pull request closed without merging**